local-dev/bin/render

```diff
@@ -2,26 +2,85 @@
+    -e "s/{{ \$secret.name }}/$CLOUDSQL_INSTANCE/" \
+    "$MYSQL_TEMPLATE_FILE" > "$MYSQL_OUTPUT_FILE"
 
-cat <<EOF
+#Files to copy directly to config##
+cp local-dev/templates/docker-rsync-local-rawls.sh config/docker-rsync-local-rawls.sh
+cp local-dev/templates/liquibase.properties config/liquibase.properties
+cp local-dev/templates/local-mysql.conf config/local-mysql.conf
+cp local-dev/templates/local-sam.conf config/local-sam.conf
+cp local-dev/templates/mod_security_ignore.conf config/mod_security_ignore.conf
+cp local-dev/templates/oauth2.conf config/oauth2.conf
+cp local-dev/templates/site.conf config/site.conf
 
-Don't forget to add the local.broadinstitute.org alias to your hosts file!
+##Files to download from secrets##
+gcloud secrets versions access latest --project broad-dsde-dev --secret rawls-sa > config/rawls-account.json
+gcloud secrets versions access latest --project broad-dsde-dev --secret rawls-sa > config/sqlproxy-service-account.json
+gcloud secrets versions access latest --project broad-dsde-dev --secret rawls-sa-pem | jq -r '."key.pem"' > config/rawls-account.pem
+gcloud secrets versions access latest --project broad-dsde-dev --secret rawls-bigquery-sa > config/bigquery-account.json
+gcloud secrets versions access latest --project broad-dsde-dev --secret rawls-billing-sa | jq -r '."key.pem"' > config/billing-account.pem
+gcloud secrets versions access latest --project broad-dsde-dev --secret buffer-client-sa-b64 | jq -r '.key' | base64 --decode > config/buffer-account.json
 
-$ sudo sh -c "echo '127.0.0.1       local.broadinstitute.org' >> /etc/hosts"
 
-EOF
+gcloud container clusters get-credentials --zone us-central1-a --project broad-dsde-dev terra-dev
+kubectl -n local-dev get secrets local-dev-cert -o 'go-template={{index .data "tls.crt"}}' | base64 --decode > config/server.crt
+kubectl -n local-dev get secrets local-dev-cert -o 'go-template={{index .data "tls.key"}}' | base64 --decode > config/server.key
+
+## Secrets to pull for rawls.conf ##
+RAWLS_OAUTH_CREDS_JSON=$(gcloud secrets versions access latest  --project broad-dsde-dev --secret rawls-oauth-creds)
+RAWLS_CONF_JSON=$(gcloud secrets versions access latest  --project broad-dsde-dev --secret rawls-conf)
+RAWLS_REFRESH_TOKEN_CRED_JSON=$(gcloud secrets versions access latest  --project broad-dsde-dev --secret refresh-token-oauth-credential)
+RAWLS_SAKEY_JSON=$(gcloud secrets versions access latest  --project broad-dsde-dev --secret rawls-sa)
+CROMWELL_SAKEY_JSON=$(gcloud secrets versions access latest  --project broad-dsde-dev --secret cromwell-sa)
+B2C_APPID_JSON=$(gcloud secrets versions access latest  --project broad-dsde-dev --secret b2c-application-id)
+LANDING_ZONE_POSTGRES_JSON=$(gcloud secrets versions access latest  --project broad-dsde-dev --secret wsm-azure-postgres-creds)
+WEB_OBJECT=$(echo "$RAWLS_REFRESH_TOKEN_CRED_JSON" | jq -r '.value | fromjson | .web')
+
+#Define some reusable variables
+APPS_DOMAIN="test.firecloud.org"
```

Comment:
I pulled these out early on in the theory that they might want to be replaced with a different value, but I've come to think there's no need for these to be variables and replacement, so maybe I can remove this part of the script and just hard code the values in the conf file.  Thoughts?